CMakeLists.txt

```diff
@@ -9,8 +9,8 @@ add_definitions(-std=c++11)
 
 add_executable(tests
     tests/tests.cpp
-    include/morphsnakes/ndimage.h
-    include/morphsnakes/narrowband.h
-    include/morphsnakes/morphsnakes.h
+    morphsnakes/include/ndimage.h
+    morphsnakes/include/narrowband.h
+    morphsnakes/include/morphsnakes.h
```

Comment:
I explicitly want the header files to be in a folder called `morphsnakes`. If someone installs the library for development, they should be installed in folders like `/usr/local/include/morphsnakes`.

Similarly, I explicitly want the #include's to be done with
```
#include <morphsnakes/morphsnakes.h>
```
This is standard in many C++ libraries (see opencv for example).